numpy/core/src/multiarray/ctors.c

```diff
@@ -3166,7 +3166,7 @@ PyArray_Zeros(int nd, npy_intp *dims, PyArray_Descr *type, int is_f_order)
  * Empty
  *
  * accepts NULL type
- * steals referenct to type
+ * steals referenced to type
```

Comment:
should this actually `reference` instead ?

Reply:
steals a reference to type

Reply:
f3ff2d6